web/src/xml.js

```diff
@@ -0,0 +1,18 @@
+const { URL, SITE_TITLE } = require('../../config.js');
+
+function getOpenSearchXml() {
+  return (
+    `<ShortName>${SITE_TITLE}</ShortName>` +
+    `<Description>Search ${SITE_TITLE}</Description>` +
+    '<InputEncoding>UTF-8</InputEncoding>' +
+    `<Image width="32" height="32" type="image/png">${URL}/public/favicon.png</Image>` +
```

Comment:
I think this is also in the config?

Reply:
What are you referring to? Image size and type?

Reply:
Looking at the source for the favicon it seems to have these properties hard-coded https://github.com/lbryio/lbry-desktop/blob/master/static/index-web.html#L7